### PR TITLE
Adds two new handlerbars helpers: 'substring' and 'includes'. (ref #246)

### PR DESCRIPTION
**Pull request by joseeesquivel**
…ee-io#246

### CODE REVIEW

File: src/handlebar-helpers.js [hunk 1]
```diff
@@ -42,7 +42,25 @@ const helpers = {
...
-  }
+  },
+  substring: function (data, startIndex, endIndex) {
+    if ( typeof data  === 'string' && typeof startIndex === 'number' && typeof endIndex === 'number') {
+      return data.substring(startIndex, endIndex);
+    }
+    if ( typeof data  === 'string' && typeof startIndex === 'number') {
+      return data.substring(startIndex);
+    }
+    return ''
```
Comment (alewitt2) [inline src/handlebar-helpers.js:53]: in javascript, its ok to pass in undefined variables to a function. lets minimize the code a bit by utilizing this behavior.
```suggestion
    if ( typeof data  === 'string' && typeof startIndex === 'number' && (typeof endIndex === 'number' || endIndex === undefined )) {
      return data.substring(startIndex, endIndex);
    }
    return ''
```

File: src/handlebar-helpers.js [hunk 2]
```diff
@@ -42,7 +42,25 @@ const helpers = {
...
+  },
+  includes: function (arr, valueToFind, fromIndex) {
+    if ( Array.isArray(arr) && valueToFind !== 'undefined' && typeof fromIndex === 'number') {
+        return arr.includes(valueToFind, fromIndex)
+    }
+    if ( Array.isArray(arr) && typeof valueToFind !== 'undefined') {
+        return arr.includes(valueToFind)
+    }
+    return false
```
Comment (alewitt2) [inline src/handlebar-helpers.js:62]: minimze code and add semicolons
```suggestion
  includes: function (arr, valueToFind, fromIndex) {
    fromIndex = ( typeof fromIndex === 'number' ) ? fromIndex : undefined;
    if ( Array.isArray(arr) && valueToFind !== 'undefined') {
        return arr.includes(valueToFind, fromIndex);
    }
    return false;
```

File: src/handlebar-helpers.js (hunk 1 above)
Comment (joseeesquivel) [reply src/handlebar-helpers.js:53]: It makes sense, updating code as suggested ....

File: src/handlebar-helpers.js (hunk 2 above)
Comment (joseeesquivel) [reply src/handlebar-helpers.js:62]: It makes sense, updating code as suggested ....

File: src/handlebar-helpers.js (hunk 1 above)
Comment (alewitt2) [reply src/handlebar-helpers.js:53]: also, make sure to update your tests according to the new return result when there are errors

**Review by alewitt2 — APPROVED** [review]

**Pull request merged**